src/cljs/rems/search.cljs

```diff
@@ -25,7 +25,7 @@
     i.e. a fetch after the initial fetch
 
   All state is stored in `db` under `::foo`, which can be removed to reset state."
-  [id url]
+  [id url & [options]]
```

Comment:
This is unused?

Reply:
yeah, leftover from an earlier experiment. removed.